delete/delete.go

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Add a Readme in this Dir on what this does and where the image as well this code used.